Assets/Scripts/StateMachines/PlushieDamageSM.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlushieDamageSM : BaseStateMachine
```

Comment:
We should put our classes in namespaces. The higher-level game manager is in the GameState namespace. You can use that one, or create a more specific one.